viewer/packages/api/src/public/createRevealManager.ts

```diff
@@ -149,7 +149,6 @@ export function createRevealManager(
     sceneHandler.scene,
     renderer
   );
-  sceneHandler.customObjects.push(pointCloudManager.pointCloudGroupWrapper);
```

Comment:
Seem to remember that this was needed and was somehow related to budget being consistent across multiple point cloud models.

Reply:
Can confirm that I see no problem with point cloud budget for multiple models.